### PR TITLE
WPF-62704-Add Additional content for the Spline types.

### PR DESCRIPTION
**Pull request by MoneeshramDhanabal**
Added the additional explanation content for the Spline types.

### CODE REVIEW

File: wpf/Charts/SeriesTypes/Spline.md [hunk 1]
```diff
@@ -154,6 +154,8 @@ N> You can explore our [WPF Spline Area Chart](https://www.syncfusion.com/wpf-co
 
 **Cardinal**
 
+The cardinal spline algorithm ensures that the resulting curve passes through each data point, while also providing a smooth and visually appealing interpolation with continuous derivatives up to a certain order at each point. This makes it a popular choice for use in computer graphics and image processing applications.
+
```
Comment (MuneeshKumarG) [inline wpf/Charts/SeriesTypes/Spline.md:158]: This makes it a popular choice for use in computer graphics and image processing applications.

Remove this line

File: wpf/Charts/SeriesTypes/Spline.md [hunk 2]
```diff
@@ -206,6 +210,12 @@ series.SplineType = SplineType.Monotonic;
 
 **Clamped**
 
+The clamped splines algorithm ensures that the resulting curve passes through the first and last data points and has a specified slope or curvature at these endpoints, thus ensuring that the curve is well-behaved at the boundaries and does not exhibit any unusual behavior. 
+
+Clamped splines are often used in engineering and scientific applications where the behavior of the curve at the endpoints is critical. 
+
+However, one important characteristic of clamped splines is that they can produce sharp bends or kinks in the curve if the data points are too far apart or if the constraints are too restrictive, resulting in an unrealistic or non-smooth curve. Therefore, it is important to choose appropriate constraints and data points when using clamped splines.
+
```
Comment (MuneeshKumarG) [inline wpf/Charts/SeriesTypes/Spline.md:218]: Clamped splines are often used in engineering and scientific applications where the behavior of the curve at the endpoints is critical. 

However, one important characteristic of clamped splines is that they can produce sharp bends or kinks in the curve if the data points are too far apart or if the constraints are too restrictive, resulting in an unrealistic or non-smooth curve. Therefore, it is important to choose appropriate constraints and data points when using clamped splines.

Remove above sections.

Comment (MoneeshramDhanabal) [reply wpf/Charts/SeriesTypes/Spline.md:218]: Removed the commented sections.

File: wpf/Charts/SeriesTypes/Spline.md (hunk 1 above)
Comment (MoneeshramDhanabal) [reply wpf/Charts/SeriesTypes/Spline.md:158]: Removed the mentioned lines.

**Review by MuneeshKumarG — APPROVED** [review]
This is fine

**Pull request merged**